content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+[Crossplane XPKG specification](https://github.com/crossplane/crossplane/blob/master/contributing/specifications/xpkg.md).
+
+The `crossplane` CLI supports building 
+[configuration]({{< ref "../concepts/packages" >}}),
+[function]({{<ref "../concepts/composition-functions">}}) and 
+[provider]({{<ref "../concepts/providers" >}}) package types. 
+
+
+#### Flags
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                            | Description                    |
+| ------------ | -------------                        | ------------------------------ |
+|              | `--embed-runtime-image-name=NAME`    |  The image name and tag of a container to include in the package. Only for provider and function packages. |
+|              | `--embed-runtime-image-tarball=PATH` |  The filename of a container to include in the package. Only for provider and function packages.                              |
+| `-e`         | `--examples-root="./examples"`       |  The path to a directory of examples related to the package.                               |
+|              | `--ignore=PATH,...`                  |  List of files and directories to ignore.                              |
+| `-o`         | `--package-file=PATH`                |  Directory and filename of the created package.                             |
+| `-f`         | `--package-root="."`                 |  Directory to search for YAML files.                              |
+{{< /table >}}
+
+The `crossplane xpkg build` command recursively looks in the directory set by 
+`--package-root` and attempts to combine any files ending in `.yml` or `.yaml` 
+into a package.
+
+All YAML files must be valid Kubernetes manifests with `apiVersion`, `kind`, 
+`metadata` and `spec` fields. 
+
+#### Ignore files
+
+Use `--ignore` to provide a list of files and directories to ignore.  
+
+For example,  
+`crossplane xpkg build --ignore="./test/*,kind-config.yaml"`
+
+#### Set the package name
+
+`crossplane` automatically names the new package a combination of the 
+`metadata.name` and a hash of the package contents and saves the contents 
+in the same location as `--package-root`. Define a specific location and 
+filename with `--package-file` or `-o`.  
+
+For example,  
+`crossplane xpkg build -o /home/crossplane/example.xpkg`.
+
+
+#### Include examples
+
+Include YAML files demonstrating how to use the package with `--examples-root`. 
+
+[Upbound Marketplace](https://marketplace.upbound.io/) uses files included with 
+`--examples-root` as documentation for published packages.
+
+#### Include a runtime image
+
+Functions and Providers require YAML files describing their dependencies and 
+settings as well as a container image for the function or provider controller.
+
+Using `--embed-runtime-image-name` downloads a specified container image and 
+includes the image inside the function or provider package. Use the 
```

Comment:
Looks like this trails off.

Also, I'm not sure it does actually download the image. I think it connects to the Docker daemon and looks for it there, so you might need to explicitly pull it first.

Reply:
I couldn't tell from the `daemon.image` docs, can you confirm that it won't download a local package?

Reply:
Yeah, confirmed you have to manually pull. For example:

```shell
$ crossplane xpkg build -f package --embed-runtime-image=alpine
crossplane: error: failed to get runtime base image options: failed to pull runtime image: Error response from daemon: No such image: alpine:latest

$ docker pull alpine
Using default tag: latest
latest: Pulling from library/alpine
579b34f0a95b: Pull complete
Digest: sha256:eece025e432126ce23f223450a0326fbebde39cdf496a85d8c016293fc851978
Status: Downloaded newer image for alpine:latest
docker.io/library/alpine:latest

$ crossplane xpkg build -f package --embed-runtime-image=alpine
$ echo $?
0
```